### PR TITLE
Replace OfficeDev/office-ui-fabric-react with microsoft/fluentui

### PR DESCRIPTION
**Pull request by ecraig12345**
Replace OfficeDev/office-ui-fabric-react with microsoft/fluentui across all files.

Remove OfficeDev/uifabric-calendar team from codeowners.

### CODE REVIEW

File: azure-pipelines.release.yml [hunk 1]
```diff
@@ -28,7 +28,7 @@ steps:
   - script: |
       git config user.name "UI Fabric Build"
       git config user.email "fabrictactical@service.microsoft.com"
```
Comment (dzearing) [inline azure-pipelines.release.yml]: fluentui-internal@service.microsoft.com

File: azure-pipelines.release-fluentui.yml [hunk 2]
```diff
@@ -19,7 +19,7 @@ steps:
   - script: |
       git config user.name "UI Fabric Build"
       git config user.email "fabrictactical@service.microsoft.com"
```
Comment (dzearing) [inline azure-pipelines.release-fluentui.yml]:
```suggestion
      git config user.email "fluentui-internal@service.microsoft.com"
```

**Comment by cliffkoh** [comment]
@ecraig12345  There is microsoft/uifabric-calendar as well,

**Comment by ecraig12345** [comment]
@cliffkoh I talked with David about this and we agreed to remove it. The group didn't cause email notifications, and John was the only calendar person reviewing stuff anyway.

**Comment by cliffkoh** [comment]
Just to provide some historical context: I think the group was set up to manage the moments when John was OOF, and to allow his teammates the "power" to unblock calendar PRs.

**Pull request merged**